### ISSUE TITLE
[Other Issues]: Krypton extended help exe is classed as "security conflicting app"

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Extended-Toolkit#161]
![image](https://user-images.githubusercontent.com/2418812/116533430-47266400-a8d9-11eb-881b-3b8d73e98e76.png)

The above will **not** be allowed to run in some Companies computers 

(i.e. "Run Anayway" will not be visible)

**Comment by Wagnerp** [comment]
Umm... I wonder why, as it only installs a `chm` file, nothing else.

**Comment by Smurf-IV** [comment]
I installed the it on another machine and I got this.

1) This is an exe not an msi
2) it installs `bat` and `exe` files as well:
![image](https://user-images.githubusercontent.com/2418812/116654915-059dc380-a982-11eb-84dc-328cd5594334.png)

**Comment by Wagnerp** [comment]
The `bat` & `exe` files are there to integrate the help files into Visual Studio. I'll make a corporate compliant `msi` option with just the `chm` file in it.

**Comment by Wagnerp** [comment]
@Smurf-IV Would you mind testing this: https://1drv.ms/u/s!AnKzT7Em-ox5g89WuHMpmERkq_f13A?e=dbQNbG

**Comment by Smurf-IV** [comment]
> 
> 
> @Smurf-IV Would you mind testing this: https://1drv.ms/u/s!AnKzT7Em-ox5g89WuHMpmERkq_f13A?e=dbQNbG

That will have to wait until Tuesday ;-)

**Comment by Smurf-IV** [comment]
I had to log onto my Work PC..
So tested this.
Same message

**Comment by Wagnerp** [comment]
Ok, I'll try creating a standard `msi` package.

**Comment by Wagnerp** [comment]
Is it possible to do this using WiX?

**Comment by Smurf-IV** [comment]
Yes. wix is used to create the installers for DevStudio, so it can be as simple, or as complicated as you like.

For simple then look @ my [Elucidate ](https://github.com/Smurf-IV/Elucidate/tree/master/Elucidate/Elucidate%20Windows%20Installer) project.

**Comment by Wagnerp** [comment]
Any suggestions about GUIs?

**Comment by Smurf-IV** [comment]
Use the default, it's easiest, and then just replace the bitmaps that are used to surround the installation text steps

**Comment by Wagnerp** [comment]
Ok, I'll look into it this week

**Comment by Wagnerp** [comment]
@Smurf-IV Possibly found the reason, it was not digitally signed.

**Comment by Smurf-IV** [comment]
That can cause that screen to appear as well, 
i.e. signing cert is old, or did not happen, or happen correctly.

**Comment by Wagnerp** [comment]
I'll be uploading a signed installer to test soon

**Comment by Wagnerp** [comment]
@Smurf-IV Hopefully this signed executable should work: https://1drv.ms/u/s!AnKzT7Em-ox5g89XXe0I0hV0W1KuOA?e=Mw4BP0

**Comment by Smurf-IV** [comment]
Same for both :-(
![image](https://user-images.githubusercontent.com/2418812/117104052-5e4ed100-ad73-11eb-8e5b-4910fd998671.png)

**Comment by Wagnerp** [comment]
@Smurf-IV Will DM you on Discord with known signed binaries
